README.md

```diff
@@ -1,10 +1,24 @@
+<div align="center">
+
 [![Orange banner indicating a preview software component][release-level-banner--unstable]](##)
 
-<br />
+<a href="##" height="289" width="996"
+	><img
+		alt="NOT Pablo Picasso and NOT Vincent van Gogh"
+		src="https://github-production-user-asset-6210df.s3.amazonaws.com/132791922/320540533-19063f11-a726-41e3-bb8d-a30342c03683.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAVCODYLSA53PQK4ZA%2F20240408%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240408T151840Z&X-Amz-Expires=300&X-Amz-Signature=7560f942de8bbf5e9e0887e942bae28ad2b3939907424a0ca3e6c1286ef5a309&X-Amz-SignedHeaders=host&actor_id=132791922&key_id=0&repo_id=525886105"
```

Comment:
```suggestion
		src="https://raw.githubusercontent.com/OpenINF/openinf.github.io/live/assets/img/png/docker-fisher-banner-top.png"
```

Reply:
we must first make necessary edits/conversion to jpg, but would rather prefer embedded svg